presto-docs/src/main/sphinx/release/release-341.md

```diff
@@ -0,0 +1,138 @@
+# Release 341 (XX Aug 2020)
+
+## General Changes
+
+* Add support for variable-precision `TIME` type. ({issue}`4381`)
+* Add Iceberg connector.
+* Add {func}`human_readable_seconds`. ({issue}`4344`)
+* Add support for {func}`extract` for `timestamp(p) with time zone` with values of `p` other than 3. ({issue}`4867`)
+* Add support for correlated subqueries in recursive queries. ({issue}`4877`)
+* Expose physical input bytes read statistics. ({issue}`4600`}
+* Report dynamic filters stats. ({issue}`4440`)
+* Improve query scalability when new nodes are added to cluster. ({issue}`4294`)
+* Reduce latency of how fast lazy dynamic filters are collected. ({issue}`4924`) 
+* Fix serialization of NULL values in structural types for old Presto clients ({issue}`4778`)
+* Fix failure when aggregation query contains duplicate expressions. ({issue}`4872`)
+* Fix compiler failure when querying long timestamp type. ({issue}`4824`)
+* Change timestamp operations to match the SQL specification. The value of a `TIMESTAMP` type is not affected by the session 
+  time zone. ({issue}`4799`)
+* Fix parsing failure of timestamps due to daylight saving changes. ({issue}`4799`)
+* Remove the `deprecated.legacy-timestamp` configuration property and the `legacy_timestamp` session property. ({issue}`4799`)
```

Comment:
Did the docs for that get removed as well?

Reply:
Yes, no references in the code base, other than the "defunct" declaration